utils/coreutils/utils_test.go

```diff
@@ -162,3 +162,19 @@ func TestListToText(t *testing.T) {
 	assert.Equal(t, ListToText([]string{"one", "two"}), "one and two")
 	assert.Equal(t, ListToText([]string{"one", "two", "three"}), "one, two and three")
 }
+
+func TestInterfaceToMap(t *testing.T) {
```

Comment:
It looks like this function only tests that the `InterfaceToMap` function doesn't return an error, and doesn't validate the content of the returned map.